discord/member.pyi

```diff
@@ -0,0 +1,114 @@
+import discord.abc
+import datetime
+
+from .activity import BaseActivity, Spotify
+from .enums import Status, DefaultAvatar
+from .colour import Colour
+from .message import Message
+from .role import Role
+from .permissions import Permissions
+from .channel import VoiceChannel, GroupChannel, DMChannel
+from .guild import Guild
+from .relationship import Relationship
+from .user import Profile, User
+
+from typing import Any, Optional, Union, List, Tuple
+
+class VoiceState:
+    deaf: bool
+    mute: bool
+    self_mute: bool
+    self_deaf: bool
+    self_stream: bool
+    self_video: bool
+    afk: bool
+    channel: Optional[Union[GroupChannel, VoiceChannel]]
+    session_id: str
+
+class Member(discord.abc.Messageable, discord.abc.User):
+    joined_at: Optional[datetime.datetime]
+    activities: Tuple[Union[BaseActivity, Spotify], ...]
+    nick: Optional[str]
+    premium_since: Optional[datetime.datetime]
+    guild: Guild
+
+    # From discord.user.BaseUser
+    name: str
+    id: int
+    discriminator: str
+    avatar: Optional[str]
+    bot: bool
+    system: bool
+
+    @property
+    def avatar_url(self) -> str: ...
+    def is_avatar_animated(self) -> bool: ...
+    def avatar_url_as(self, *, format: Optional[str] = ..., static_format: str = ...,
+                      size: int = ...) -> str: ...
+    @property
+    def default_avatar(self) -> DefaultAvatar: ...
+    @property
+    def default_avatar_url(self) -> str: ...
```

Comment:
This returns `Asset` object, like in its superclass `discord.user.BaseUser`
```suggestion
    def default_avatar_url(self) -> Asset: ...
```

Reply:
Fixed